android/build.sh

```diff
@@ -93,42 +101,38 @@ function build_openblas {
-        SYSROOT=$NDK_ROOT/platforms/android-$ANDROID_NATIVE_API_LEVEL/arch-arm64
+        GCC_TOOLCHAIN=$NDK_ROOT/toolchains/aarch64-linux-android-4.9/prebuilt/$HOST_OS-$HOST_BIT
+        LLVM_TOOLCHAIN=$NDK_ROOT/toolchains/llvm/prebuilt/$HOST_OS-$HOST_BIT
+        SYSROOT=$NDK_ROOT/platforms/$ANDROID_PLATFORM/arch-arm64
+        CLANG_FLAGS="-target aarch64-linux-android --sysroot $SYSROOT -gcc-toolchain $GCC_TOOLCHAIN"
+        OPENBLAS_LDFLAGS="-L$GCC_TOOLCHAIN/lib/gcc/aarch64-linux-android/4.9.x -lm"
         TARGET=ARMV8
-        BINARY=64
-    elif [ "$ANDROID_ABI" = "armeabi" ]; then
-        CROSS_SUFFIX=$NDK_ROOT/toolchains/arm-linux-androideabi-4.9/prebuilt/$HOST_OS-$HOST_BIT/bin/arm-linux-androideabi-
-        SYSROOT=$NDK_ROOT/platforms/android-$ANDROID_NATIVE_API_LEVEL/arch-arm
-        TARGET=ARMV5
-        BINARY=32
-    elif [ "$ANDROID_ABI" = "x86" ]; then
-        CROSS_SUFFIX=$NDK_ROOT/toolchains/x86-4.9/prebuilt/$HOST_OS-$HOST_BIT/bin/i686-linux-android-
-        SYSROOT=$NDK_ROOT/platforms/android-$ANDROID_NATIVE_API_LEVEL/arch-x86
-        TARGET=ATOM
-        BINARY=32
-    elif [ "$ANDROID_ABI" = "x86_64" ]; then
-        CROSS_SUFFIX=$NDK_ROOT/toolchains/x86_64-4.9/prebuilt/$HOST_OS-$HOST_BIT/bin/x86_64-linux-android-
-        SYSROOT=$NDK_ROOT/platforms/android-$ANDROID_NATIVE_API_LEVEL/arch-x86_64
-        TARGET=ATOM
-        BINARY=64
+    elif [ "$ANDROID_ABI" = "armeabi-v7a" ]; then
+        GCC_TOOLCHAIN=$NDK_ROOT/toolchains/arm-linux-androideabi-4.9/prebuilt/$HOST_OS-$HOST_BIT
+        LLVM_TOOLCHAIN=$NDK_ROOT/toolchains/llvm/prebuilt/$HOST_OS-$HOST_BIT
+        SYSROOT=$NDK_ROOT/platforms/$ANDROID_PLATFORM/arch-arm
+        CLANG_FLAGS="-target arm-linux-androideabi -marm -mfpu=neon -mfloat-abi=softfp --sysroot $SYSROOT -gcc-toolchain $GCC_TOOLCHAIN"
+        OPENBLAS_LDFLAGS="-L$GCC_TOOLCHAIN/lib/gcc/arm-linux-androideabi/4.9.x"
+        TARGET=ARMV7
     else
         echo "Unsupport Android ABI: $ANDROID_ABI"
         exit 1
     fi
     cd $OPENBLAS_BUILD_ROOT
-    # make -j$ANDROID_BUILD_JOBS \
-    #      CC="${CROSS_SUFFIX}gcc --sysroot=$SYSROOT" \
-    #      FC="${CROSS_SUFFIX}gfortran --sysroot=$SYSROOT" \
-    #      CROSS_SUFFIX=$CROSS_SUFFIX \
-    #      HOSTCC=gcc USE_THREAD=1 NUM_THREADS=4 USE_OPENMP=1 \
-    #      NO_LAPACK=1 TARGET=$TARGET BINARY=$BINARY
-    make -j$ANDROID_BUILD_JOBS \
-         CC="${CROSS_SUFFIX}gcc --sysroot=$SYSROOT" \
-         NOFORTRAN=1 \
-         CROSS_SUFFIX=$CROSS_SUFFIX \
-         HOSTCC=gcc USE_THREAD=1 NUM_THREADS=4 \
-         NO_LAPACK=1 TARGET=$TARGET BINARY=$BINARY
+
+    # save current env
+    OLD_PATH=$PATH
+    OLD_LDFLAGS=$LDFLAGS
+
+    export PATH=$GCC_TOOLCHAIN/bin:$LLVM_TOOLCHAIN/bin:$PATH
+    export LDFLAGS=$OPENBLAS_LDFLAGS
+
+    make TARGET=$TARGET ONLY_CBLAS=1 AR=ar CC="clang $CLANG_FLAGS" HOSTCC=gcc ARM_SOFTFP_ABI=1 -j$ANDROID_BUILD_JOBS
```

Comment:
build failed on my windows PC in this line. My NDK version is NDK16.1.4479499(get this information from Android Studio), does this version satisfies the requirements? The error message
```
C Compiler (clang -target aarch64-linux-android --sysroot D:/3rdparty/Android/sdk/ndk-bundle/platforms/android-21/arch-arm64 -gcc-toolchain D:/3rdparty/Android/sdk/ndk-bundle/toolchains/aarch64-linux-android-4.9/prebuilt/windows-x86_64) is something wrong.
```

Something related to Clang?